src/sorters/abstract.py

```diff
@@ -1,22 +1,37 @@
+from typing import Optional, List
+from pathlib import Path
+
 from abc import ABC, abstractmethod
 import yaml
+from PIL import Image
 
 
 class AbstractSorter(ABC):
 
-    def __init__(self, input_path: str, output_path: str, config_path: str):
+    def __init__(self, input_path: str, output_path: str, config_path: Optional[str] = None):
         self.input_path = input_path
+        self.file_names = self.get_file_names(self.input_path)
```

Comment:
It would be better not to change Abstract implementation, but in new class __init__ implementation

Reply:
Because, maybe we don't need this functional